server/apps/main/views.py

```diff
@@ -257,7 +257,7 @@ def share_experience(request, uuid=False):
             )
 
     else:
-        return redirect("index")
+        return registration(request)
```

Comment:
weird, this new redirect does not work for me!

Reply:
That is weird. What's the behaviour you're seeing? Does it go back to the home page instead?

The expected behaviour is that it doesn't redirect at all, but rather serves the Registration Page html content while retaining the `add_story` url.

Reply:
Yeah, so for me it just took me back to the `home` page instead of loading the registration page!

Reply:
Strangely, I can't reproduce this behaviour now, instead it correctly redirects.  🤔 Maybe there was just some caching problem or a problem that resolved itself by completely restarting the docker! so nevermind on this one!